### PR TITLE
GFG PODT 2 nov 24 sol. in py

### PR DESCRIPTION
**Pull request by thedarkking01**
@Ayu-hack Hii 😊
pic: 
image
image

### CODE REVIEW

**Comment by Ayu-hack** [comment]
you are providing solution for 2 nov but you have incorrectly name the code file as 2 oct and merging it in OCT FOLDER. Please correct it and create new PR

**Comment by Ayu-hack** [comment]
It is GFG POTD not GFG PODT please correct next time n be careful

File: October 2024/Kth distance(2 oct 24).py
Comment (Ayu-hack) [inline October 2024/Kth distance(2 oct 24).py:1]: this is not a solution for 2 nov.

Comment (thedarkking01) [reply October 2024/Kth distance(2 oct 24).py:1]: i correct the file name

**Pull request closed without merging**